src/main/webapp/resources/css/opd_five_five.css

```
@@ -106,6 +106,53 @@
+
+/* Bill No */
+    .paymentTable th:nth-child(1),
+    .paymentTable td:nth-child(1) {
+    width: 180px;
+    }
+
+/* Service */
+    .paymentTable th:nth-child(2),
+    .paymentTable td:nth-child(2) {
+    width: auto;
+    word-wrap: break-word;
```

Stylelint (17.14.1)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)